pkg/proc/native/proc_linux.go

```diff
@@ -45,6 +47,14 @@ const (
 // process details.
 type osProcessDetails struct {
 	comm string
+
+	ctx *ebpf.EBPFContext
```

Comment:
Given where this is place it should have a more descriptive name, like ebpfCtx or just ebpf.

Reply:
Done.